konnectd/CHANGELOG.md

```diff
@@ -0,0 +1,151 @@
+# Changelog for [0.3.2] (2020-07-23)
+
+The following sections list the changes in ocis-konnectd 0.3.2.
+
+[0.3.2]: https://github.com/owncloud/ocis/konnectd/compare/v0.3.1...v0.3.2
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `142`)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)